tests/introspection/tests.py

```diff
@@ -171,8 +171,8 @@ def test_get_indexes(self):
         with connection.cursor() as cursor:
             indexes = connection.introspection.get_indexes(cursor, Article._meta.db_table)
         if connection.features.can_introspect_index_type:
-            self.assertIn('type', indexes['reporter_id'])
-            indexes['reporter_id'].pop('type')
+            index_type = indexes['reporter_id'].pop('type', None)
+            self.assertEqual(index_type, 'btree')
```

Comment:
Yes, but I thought Django's introspection layer could handle that. For example, if Oracle returns "BT" for btree indexes `oracle.introspection` can convert it to "btree". Else we would end up doing that in the tests which would be ugly.